### ISSUE TITLE
ConvLSTM2D with CUDNN crashes

### ISSUE THREAD

**Issue by sivi299** [tensorflow/tensorflow#46820]
System Information: Windows 10
tensorflow version: 2.5.0-dev20201217
python version: 3.7.9
cuda version : 10.2

On using the above model, python crashes after

`model = models.Sequential(
	[
		layers.Input(
			shape=(timesteps, width, height, channels)
		),
		layers.ConvLSTM2D(
			filters=32, kernel_size=(3, 3), padding="same", return_sequences=True, dropout=0.1, recurrent_dropout=0.1
		),
		layers.MaxPool3D(
			pool_size=(1, 2, 2), strides=(1, 2, 2), padding="same"
		),
		layers.BatchNormalization(),
		layers.ConvLSTM2D(
			filters=16, kernel_size=(3, 3), padding="same", return_sequences=True, dropout=0.1, recurrent_dropout=0.1
		),
		layers.MaxPool3D(
			pool_size=(1, 2, 2), strides=(1, 2, 2), padding="same"
		),
		layers.BatchNormalization(),
		layers.ConvLSTM2D(
			filters=8, kernel_size=(3, 3), padding="same", return_sequences=False, dropout=0.1, recurrent_dropout=0.1
		),
		layers.MaxPool2D(
			pool_size=(2, 2), strides=(2, 2), padding="same"
		),
		layers.BatchNormalization(),
		layers.Flatten(),
		layers.Dense(192, activation='relu'),
		layers.Dense(action_num, activation='softmax')
	]
)
`
On using the above model, python crashes after

>2021-02-01 10:03:12.227916: I tensorflow/compiler/mlir/mlir_graph_optimization_pass.cc:127] None of the MLIR optimization passes are enabled (registered 2)
Epoch 1/300
2021-02-01 10:03:21.950076: I tensorflow/stream_executor/platform/default/dso_loader.cc:49] Successfully opened dynamic library cudnn64_8.dll
2021-02-01 10:03:22.694349: I tensorflow/stream_executor/cuda/cuda_dnn.cc:334] Loaded cuDNN version 8005
2021-02-01 10:03:23.281554: I tensorflow/core/platform/windows/subprocess.cc:308] SubProcess ended with return code: 0

>2021-02-01 10:03:23.769410: I tensorflow/core/platform/windows/subprocess.cc:308] SubProcess ended with return code: 0

>2021-02-01 10:03:23.867774: I tensorflow/stream_executor/platform/default/dso_loader.cc:49] Successfully opened dynamic library cublas64_11.dll
2021-02-01 10:03:24.624172: I tensorflow/stream_executor/platform/default/dso_loader.cc:49] Successfully opened dynamic library cublasLt64_11.dll

on further examining the system logs found the following

>Faulting application name: python.exe, version: 3.7.9150.1013, time stamp: 0x5f3ad38e
Faulting module name: _pywrap_tensorflow_internal.pyd, version: 0.0.0.0, time stamp: 0x5fdb260c
Exception code: 0xc00000fd
Fault offset: 0x00000000094ae988
Faulting process id: 0x35dc
Faulting application start time: 0x01d6f85196812501
Faulting application path: C:\Users\Scorp\AppData\Local\Programs\Python\Python37\python.exe
Faulting module path: C:\Users\Scorp\AppData\Local\Programs\Python\Python37\lib\site-packages\tensorflow\python\_pywrap_tensorflow_internal.pyd

keras config

{
    "floatx": "float32",
    "epsilon": 1e-07,
    "backend": "tensorflow",
    "image_data_format": "channels_last"
}

PFA the code and files

https://github.com/sivi299/lstm

**Comment by amahendrakar** [comment]
@sivi299,
TensorFlow v2.4 and TF-nightly are compatible with **CUDA 11.0** and **cuDNN 8**. For more information, please take a look at the [tested build configurations](https://www.tensorflow.org/install/source_windows#gpu).

I was able to run the code on TF v2.4 without any issues, please find the gist of it [here](https://colab.research.google.com/gist/amahendrakar/02f234c0dfcfbc44ff512a2d00422285/46820.ipynb). Thanks!

**Comment by sivi299** [comment]
@amahendrakar thanks for the quick reply, I did try with Cuda 11 I am still facing the same issue I am not sure if its something to do with the GPU model itself, I am using an Nvidia 3090 series

**Comment by sivi299** [comment]
I removed tf-nightly and installed tensorflow2.4 and it works fine now thanks!
